gaming/servers/create_cluster.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
```

Comment:
```suggestion
// Copyright 2020 Google LLC
```

Applies to all of the new files.

Reply:
done.